solr/core/src/java/org/apache/solr/handler/admin/SplitOp.java

```diff
@@ -189,6 +205,7 @@ public void execute(CoreAdminHandler.CallInfo it) throws Exception {
       parentCore.getUpdateHandler().split(cmd);
 
       if (it.handler.coreContainer.isZooKeeperAware()) {
+        log.trace("SplitOp: Create cloud descriptors for sub-shards of core: {}", cname);
```

Comment:
core is redundant with MDC?

Reply:
i think logging the steps what is happening is not redundant with the corresponding core name.